seminaire-saint-sulpice-ecole-theologie.csl

```diff
@@ -505,15 +506,21 @@
     <label plural="never" suffix=" " variable="page" form="short"/>
     <text variable="locator" form="short"/>
   </macro>
-  <citation et-al-min="4" et-al-use-first="3" et-al-subsequent-min="3" et-al-subsequent-use-first="1">
-    <layout suffix="." delimiter="&#160;; ">
+  <macro name="number-citation">
+    <label plural="never" prefix="n°" suffix=" " variable="number" form="short"/>
```

Comment:
This seems weird -- what are you trying to do here?

Reply:
I would like that if 4 quotes from the same book or article follow each other, the fourth can use the references of the quote. But I realize that this is not useful when reading other books.

Reply:
Could you give an example? And have you tested this? What this currently does is to add the number label depending on the number variable (which may not even exist for a given item) before the locator.  It seems like that can't be what you're actually trying to do.